concent_api/utils/api_view.py

```diff
@@ -22,11 +24,15 @@ def api_view(view):
     def wrapper(request, *args, **kwargs):
         if 'HTTP_CONCENT_CLIENT_PUBLIC_KEY' not in request.META or request.META['HTTP_CONCENT_CLIENT_PUBLIC_KEY'] == '':
             return JsonResponse({'error': 'Concent-Client-Public-Key HTTP header is missing on the request.'}, status = 400)
-
         try:
             client_public_key = b64decode(request.META['HTTP_CONCENT_CLIENT_PUBLIC_KEY'].encode('ascii'), validate=True)
         except binascii.Error:
             return JsonResponse({'error': 'The value in the Concent-Client-Public-Key HTTP is not a valid base64-encoded value.'}, status = 400)
+        if 'HTTP_REQUESTOR_PUBLIC_KEY' in request.META:
+            try:
+                b64decode(request.META['HTTP_REQUESTOR_PUBLIC_KEY'].encode('ascii'))
+            except TypeError:
+                return JsonResponse({'error': 'The value in the Requestor-Public-Key HTTP is not a valid base64-encoded value.'}, status = 400)
```

Comment:
You changed it to `Reqestor-Public-Key`. It was supposed to be called `Concent-Requestor-Public-Key`. With a `Concent-` prefix like all our custom HTTP headers.